src/StdUtils.sol

```diff
@@ -186,6 +170,11 @@ abstract contract StdUtils {
 
     // Used to prevent the compilation of console, which shortens the compilation time when console is not used elsewhere.
 
+    function console2_log(string memory p0) private view {
```

Comment:
Looks like this fails to compile because `StdCheats.sol` has a function with the same signature, so it clashes when both this contract and `StdCheats` are inherited from in `Test.sol`. Perhaps we namespace all of these internal `console2_log` functions, e.g. this one becomes `function console2_log_StdUtils(string memory p0) private view`. Open to other suggestions too

Reply:
sgtm